instrumentation/action_pack/test/test_helpers/controllers/example_controller.rb

```diff
@@ -7,6 +7,9 @@
 class ExampleController < ActionController::Base
   include ::Rails.application.routes.url_helpers
 
+  # Enable CSRF Protection
+  protect_from_forgery with: :exception
```

Comment:
Hi @rhythmdesai404! Since I haven't heard back from you after Eric's comment earlier, I'm assuming this is ready for review. 

I'm curious about why we need to turn this on. This is test code, and shouldn't impact the user. Why do we need to enable it here?